### DISCUSSION TITLE
Better formatted so-status

### DISCUSSION THREAD

**Discussion by netsecninja** [Security-Onion-Solutions/securityonion#1769] [Ideas] [answered]
By default, so-status has a blank line between each status, regardless of the terminal resolution. This makes the output difficult to read.

![image](https://user-images.githubusercontent.com/10831043/97804658-8f0c4d00-1c0e-11eb-82f5-65188c0b6751.png)

I found if I modified the PADDING_CONSTANT in the print_line() function from 14 to 15, the extra line was omitted, making for a much better output.

![image](https://user-images.githubusercontent.com/10831043/97804758-18238400-1c0f-11eb-9139-1d613a08c027.png)

I was unable to find that file in the git repo, so I can't submit a PR to suggest this fix. Please consider this for the next update.

**Comment by dougburks** [comment, marked as answer]
We already have an issue for this:
https://github.com/Security-Onion-Solutions/securityonion/issues/1536
